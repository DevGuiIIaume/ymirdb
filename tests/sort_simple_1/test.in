set aaaa -10 -3 0 -9 15 32 79 18 3
get aaaa
sort aaaa
list entries
bye
