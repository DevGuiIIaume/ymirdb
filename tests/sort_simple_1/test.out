> ok

> [-10 -3 0 -9 15 32 79 18 3]

> ok

> aaaa [-10 -9 -3 0 3 15 18 32 79]

> bye
